x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/WaitForRolloverReadyStep.java

```diff
@@ -113,7 +119,8 @@ public void evaluateCondition(IndexMetaData indexMetaData, Listener listener) {
                 "index [%s] is not the write index for alias [%s]", indexMetaData.getIndex().getName(), rolloverAlias)));
         }
 
-        RolloverRequest rolloverRequest = new RolloverRequest(rolloverAlias, null);
+        RolloverRequest rolloverRequest = new RolloverRequest(rolloverAlias, null)
+            .masterNodeTimeout(LifecycleSettings.LIFECYCLE_STEP_MASTER_TIMEOUT_SETTING.get(settings));
```

Comment:
can this use the `getMasterTimeout` method instead?

Reply:
I pass now timeout value directly from caller (which uses `getMasterTimeout`